README.md

```diff
@@ -241,12 +241,33 @@ python krr.py simple --help
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+<!-- Port-forwarding -->
+
+## Example of using port-forward for Prometheus
+
+If your prometheus is not auto-connecting, you can use `kubectl port-forward` for manually forwarding Prometheus.
+
+For example, if you have prometheus Pod called `kube-prometheus-st-prometheus-0`, then run this command to port-forward it:
+```sh
+kubectl port-forward pod/kube-prometheus-st-prometheus-0 9090
+```
+
+Then, run another terminal and run robusta in it, adding explicit prometheus url:
```

Comment:
Suggest a slight change for clarity:

*Then, open another terminal and run krr in it, giving an explicit prometheus url:*